dublast/panels.py

```diff
@@ -0,0 +1,208 @@
+        layout.use_property_split = True
+        layout.use_property_decorate = False  # No animation.
+        # Add settings for the current scene
+        playblast_settings = bpy.context.scene.playblast
+
+        layout.prop( playblast_settings, "resolution_percentage", slider = True )
+        layout.prop( playblast_settings, "file_format" )
+        if playblast_settings.file_format == 'PNG':
+            layout.prop( playblast_settings, "color_mode" )
+            layout.prop( playblast_settings, "compression", slider = True )
+        else:
+            layout.prop( playblast_settings, "color_mode_no_alpha" )
+            layout.prop( playblast_settings, "quality", slider = True )
+
+        #layout.use_property_split = False
+        layout.prop(playblast_settings, "use_scene_path")
+        col = layout.column()
+        col.enabled = False
+        if not playblast_settings.use_scene_path:
+            col.enabled = True
+        col.prop( playblast_settings, "filepath" )
+        col.prop( playblast_settings, "use_scene_name")
+
+class DUBLAST_PT_Metadata( Panel ):
+    bl_label = ""
+    bl_space_type = "PROPERTIES"
+    bl_region_type = "WINDOW"
+    bl_context = "output"
+    bl_parent_id = "DUBLAST_PT_Playblast_Settings"
+    bl_options = {'DEFAULT_CLOSED'}
+
+    def draw_header(self,context):
+        self.layout.label(text="Metadata", icon="INFO")
+
+    def draw(self, context):
+        layout = self.layout
+        layout.use_property_split = True
+        layout.use_property_decorate = False  # No animation.
+        playblast_settings = bpy.context.scene.playblast
+        
+        layout.prop( playblast_settings, "auto_size_stamp_font" )
+
+        row = layout.row()
+        row.enabled = not playblast_settings.auto_size_stamp_font
+        row.prop( playblast_settings, "font_size" )
+
+        layout.separator()
+
+        if bpy.app.version[0] >= 2 and bpy.app.version[1] >= 90:
+            col = layout.column(heading="Include")
+        else:
+            col = layout
+
+        metadata = [
+            "use_stamp_camera",
+            "use_stamp_date",
+            "use_stamp_filename",
+            "use_stamp_frame",
+            "use_stamp_frame_range",
+            "use_stamp_hostname",
+            "use_stamp_lens",
+            "use_stamp_marker",
+            "use_stamp_memory",
+            "use_stamp_render_time",
+            "use_stamp_scene",
+            "use_stamp_time",
+            "use_stamp_note",
+        ]
+
+        col = layout.column()
+        row = None
+        for i,m in enumerate(metadata):
+            if i % 3 == 0:
+                row = col.split(factor = 0.33)
+            row.enabled = playblast_settings.use_stamp 
+            row.prop( playblast_settings, m )
+            if m == "use_stamp_note" and playblast_settings.use_stamp_note == True:
+                col.enabled = playblast_settings.use_stamp
+                col.prop( playblast_settings, "stamp_note_text" )
```

Comment:
Three columns are too wide, check with just two if it's better or revert to a single column, that'll be better for smaller screens/panels

![image](https://user-images.githubusercontent.com/8526542/174755749-d33fcee2-1718-4e13-bd67-1aecdff542af.png)